src/test/java/co/rsk/federate/signing/hsm/message/PowHSMSignerMessageBuilderTest.java

```diff
@@ -68,12 +70,14 @@ class PowHSMSignerMessageBuilderTest {
         bridgeMainnetConstants.getBtcParams(), 9);
     private static final Federation oldFederation = TestUtils.createFederation(
         bridgeMainnetConstants.getBtcParams(), 9);
-
+    private final BtcECKey signerBtcPk = BtcECKey.fromPrivate(
+        org.bouncycastle.util.encoders.Hex.decode("fa01"));
```

Comment:
Why not use the `decode` function from `org.spongycastle.util.encoders.Hex` here?

Reply:
Good catch!